client-report/src/components/app.js

```diff
@@ -10,17 +10,17 @@ import _ from "lodash";
 
 import * as globals from "./globals";
 import DataUtils from "../util/dataUtils";
-import Matrix from "./correlationMatrix/matrix";
+// import Matrix from "./correlationMatrix/matrix";
 import Heading from "./framework/heading";
 import Footer from "./framework/Footer";
 import Overview from "./overview";
 import MajorityStrict from "./lists/majorityStrict";
 import Uncertainty from "./lists/uncertainty";
 import AllCommentsModeratedIn from "./lists/allCommentsModeratedIn"
 import ParticipantGroups from "./lists/participantGroups";
-import CommentsGraph from "./commentsGraph/commentsGraph";
+// import CommentsGraph from "./commentsGraph/commentsGraph";
 import ParticipantsGraph from "./participantsGraph/participantsGraph";
-import BoxPlot from "./boxPlot/boxPlot";
+// import BoxPlot from "./boxPlot/boxPlot";
```

Comment:
I know we're not using correlation matrix or comments graph anymore, but are we not still using boxPlot for the report?

Reply:
Looked, and indeed does not seem used.